unittests/errors/pan/functions/host_domain_from_object3.pan

```diff
@@ -0,0 +1,6 @@
+object template host_domain_from_object3;
+
+include 'pan/functions';
+
+@{fails with invalid shorthostname due to _ in name}
```

Comment:
This is excellent stuff! We should definitely be able to start using this framework. One more test I can think of, for npush - making sure that a duplicate key still fails. 

```
"/a" = dict('a', true);
"/a" = npush(''a", false);
```